engine/runtime-instrument-common/src/main/scala/org/enso/interpreter/instrument/job/ProgramExecutionSupport.scala

```diff
@@ -483,8 +483,13 @@ object ProgramExecutionSupport {
       Either
         .catchNonFatal {
           ctx.executionService.getLogger.log(
-            Level.FINE,
-            s"Executing visualization ${visualization.expressionId}"
+            Level.FINEST,
+            "Executing visualization [{0}] on expression [{1}] of [{2}]...",
+            Array[Object](
+              visualization.config,
+              expressionId,
+              expressionValue.getClass
```

Comment:
Better to use `TypeOfNode.getUncached().execute(expressionValue)` to get user friendly type, not the implementation type.

Reply:
How do the log messages look now with 2e8de06? Do they still include 
```
...of [class org.enso.interpreter.runtime.data.vector.Vector$Generic]...
```

Reply:
This is an example of an affected log message on `enso-linux-2023.2.1-nightly.2023.9.18.AppImage`:
```log
[TRACE] [2023-09-18T11:43:49+02:00] [enso] Executing visualization [VisualizationConfiguration(a910f939-ecc7-4092-ada2-7944dd72abbd,ModuleMethod(MethodPointer(Standard.Visualization.Widgets,Standard.Visualization.Widgets,get_widget_json),Vector(.row_count, ['self'])),local.New_Project_20.Main)] on expression [871380ca-7fb7-45ea-a5ec-7176029a8dd0] of [Table]...
```